components/script/dom/htmltableelement.rs

```diff
@@ -51,6 +54,31 @@ impl HTMLTableElement {
 }
 
 impl HTMLTableElementMethods for HTMLTableElement {
+    // https://html.spec.whatwg.org/multipage/#dom-table-rows
+    fn Rows(&self) -> Root<HTMLCollection> {
+        #[allow(moved_no_move)]
```

Comment:
We can store a regular `Vec<JS<Node>>` in the filter, since it's reachable by the GC.